### PR TITLE
Add decision on supporting DjangoCon Africa

### PR DESCRIPTION
**Pull request by rebkwok**

### CODE REVIEW

File: source/decisions/2023-09-25-support-djangocon-africa.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+2023-03-04 Support of DjangoCon Africa 2023
```
Comment (doismellburning) [inline source/decisions/2023-09-25-support-djangocon-africa.rst:1]:
```suggestion
2023-09-25 Support of DjangoCon Africa 2023
```

File: source/decisions/2023-09-25-support-djangocon-africa.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+===============================================
```
Comment (doismellburning) [inline source/decisions/2023-09-25-support-djangocon-africa.rst:2]:
```suggestion
===========================================
```

File: source/decisions/2023-09-25-support-djangocon-africa.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+Resolution
+----------
+
+The decision was made to support DjangoCon Africa 2023 to a value equivalent to 
+Silver level sponsorship.
+
+The exact mechanism of doing so (donation, grant, sponsorship) is considered an 
+implementation detail to be determined by UKPATL.
```
Comment (doismellburning) [inline source/decisions/2023-09-25-support-djangocon-africa.rst:18]: I don't consider this a UKPATL matter at all - purely a UKPA one - can you elaborate on where you're coming from please?

File: source/decisions/2023-09-25-support-djangocon-africa.rst [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+DjangoCon Africa takes place on 6-11 of November 2023. On 7 September 2023 the DjangoCon
+Africa Chair contacted the trustees by email to request monetary support for the event.
+
+
+Resolution
+----------
+
+The decision was made to support DjangoCon Africa 2023 to a value equivalent to 
+Silver level sponsorship.
```
Comment (doismellburning) [inline source/decisions/2023-09-25-support-djangocon-africa.rst:15]:
```suggestion
their Silver level sponsorship tier ($2250 per https://2023.djangocon.africa/static/documents/djcafrica23prospectus.pdf).
```

File: source/decisions/2023-09-25-support-djangocon-africa.rst [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+The exact mechanism of doing so (donation, grant, sponsorship) is considered an
```
Comment (doismellburning) [inline source/decisions/2023-09-25-support-djangocon-africa.rst:17]: We implicitly consider anything outside of the scope of the decision itself an implementation detail - I'm not sure this aspect needs explicitly calling out any more than e.g. whether we're paying by cheque or bank transfer - what do you think about dropping this?

**Review by doismellburning — CHANGES_REQUESTED** [review]
In spirit I approve - would like to just briefly talk details first please!

File: source/decisions/2023-09-25-support-djangocon-africa.rst [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+----------
+
+DjangoCon Africa takes place on 6-11 of November 2023. On 7 September 2023 the DjangoCon
+Africa Chair contacted the trustees by email to request monetary support for the event.
+
+
+Resolution
+----------
+
+The decision was made to support DjangoCon Africa 2023 to a value equivalent to
```
Comment (meatballs) [inline source/decisions/2023-09-25-support-djangocon-africa.rst:14]: I think this needs rewording slightly to match our previous resolution decisions:
```suggestion
To support DjangoCon Africa 2023 to a value equivalent to 
```

File: source/decisions/2023-09-25-support-djangocon-africa.rst (hunk 3 above)
Comment (rebkwok) [reply source/decisions/2023-09-25-support-djangocon-africa.rst:18]: I'd thought that the funds would be coming from UKPATL, but if that's not the case, happy to remove this bit altogether, given you other comment about implementation details.

**Review by doismellburning — APPROVED** [review]

**Review by meatballs — APPROVED** [review]

**Comment by doismellburning** [comment]
To be explicit, before I merge this:

* Daniele's involvement in DCAF (https://2023.djangocon.africa/team) would mean any involvement by him in the approval of a decision here would represent a conflict of interest
* However the 3 other trustees approve of this (me and Owen as indicated by approving reviews, Becky implicitly as author)
* This means "the majority of all of the charity trustees has signified agreement to the resolution" per clause 17 of the constitution
* I am thus merging this

**Pull request merged**